package.json

```diff
@@ -29,6 +30,7 @@
   },
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.15.5",
+    "@aurodesignsystem/auro-library": "^2.2.2",
```

Comment:
This needs to be a root dependency.  https://github.com/AlaskaAirlines/auro-accordion/commit/9e23e9fbacadbf403bf4b7ea6d6a4d1b32a5b80b